# Genome scale metabolic network model (GEM) {.unnumbered}

Metabolism is a fundamental aspect of human cellular function, and its disruption is closely linked to various health conditions, including cancer, obesity, and diabetes. Viral infections are also known to significantly alter host metabolism by hijacking and reprogramming metabolic pathways to support viral replication. Despite major technological advancements that allow measurement of thousands of metabolites, achieving a holistic understanding of cellular metabolism remains a considerable challenge. One reason for this difficulty is that metabolic activity is best defined by **metabolic fluxes**, the rates at which reactions occur, rather than by static concentrations of biomolecules such as proteins, metabolites, or mRNAs. Concentration data provide only indirect indicators of metabolic activity.

To better capture and analyze these dynamic processes, researchers use **genome-scale metabolic network models (GEMs)** which are mathematical representations of the complete metabolic network within a cell, tissue, or organism. These models describe all known metabolic reactions as a set of stoichiometric equations that can be used to simulate cellular phenotypes and predict metabolic responses to different physiological or environmental conditions. Over the past decade, numerous GEMs of human metabolism have been developed and refined, forming the foundation for studying complex biological systems and disease mechanisms.

Early efforts in this field led to the development of landmark models such as **Recon** and the **Edinburgh Human Metabolic Network (EHMN)**, which laid the groundwork for subsequent iterations including the Recon and Human Metabolic Reaction (HMR) series. These models have been instrumental in exploring disease-related metabolic alterations in conditions such as cancer, dysbiosis, fatty liver disease, and diabetes.

Despite substantial progress, challenges remain in the ongoing development and integration of human GEMs. Common issues include inconsistent use of reaction, gene, and metabolite identifiers; propagation of legacy errors from earlier models; fragmented development efforts across different research groups; and limited transparency and coordination within the modeling community. Continued collaboration and standardization are essential for refining these models and fully harnessing their potential to advance our understanding of human metabolism and its role in health and disease.

## Human 1 (Human-GEM)

Robinson and colleagues developed **Human1**, the first integrated model unifying the major human genome-scale metabolic model (GEM) lineages into a single framework known as the **Human-GEM**. This model was created through extensive curation and integration of information from the Recon and HMR model families, along with data from HMR2, iHsa, and Recon3D. The resulting Human-GEM encompasses **13,417 reactions**, **10,138 metabolites** (including **4,164 unique compounds**), and **3,625 genes**.

A version-controlled Git repository was used throughout the modelâ€™s development to ensure transparency, reproducibility, and collaboration within the scientific community. During curation, the developers removed **8,185 duplicated reactions** and **3,215 duplicated metabolites**, revised the chemical formulas of **2,016 metabolites**, rebalanced **3,226 reactions** based on stoichiometric principles, corrected the reversibility of **83 reactions**, and eliminated **576 inconsistent reactions** that violated mass conservation. Additionally, a new, updated **generic human biomass reaction** was formulated using data from various tissue and cell compositions to improve flux simulation accuracy.

To ensure model quality, the team used **Metabolic Model Testing (Memote)**, a community-driven platform for evaluating GEMs. Human1 demonstrated excellent performance in these assessments, showing **100% stoichiometric consistency**, **98.2% charge balance**, and **99.4% mass balance** across reactions.

Beyond its structural refinements, Human1 enables the integration of multiple molecular data types for comprehensive metabolic analysis. Information from databases such as CORUM, iHsa, and Recon3D, covering mammalian protein complexes and enzyme associations was integrated with gene-reaction data from earlier models to derive detailed **gene-reaction rules**. The inclusion of transcript- and protein-level associations further allows for **direct integration of omics data** into the model, facilitating context-specific simulations and deeper interpretation of cellular metabolism [@knuth84].
